src/sentry/models/pullrequest.py

```diff
@@ -99,6 +100,7 @@ class PullRequestComment(Model):
     created_at = models.DateTimeField()
     updated_at = models.DateTimeField()
     group_ids = DjangoArrayField(BoundedBigIntegerField())
+    reactions = JSONField(null=True)
```

Comment:
Is json the right choice here? Should we instead have a table we split out that contains the reactions?

Reply:
i'm storing the values from the "reactions" key for instance in this response https://api.github.com/repos/getsentry/sentry/issues/49627